metabase/metabase
# [Cmd Palette] Do not show admin and settings links to users who do not have access

Closes https://github.com/metabase/metabase/issues/49729

### Description

The above issue points out that users who are not admins were seeing settings links when they did not have access to the settings pages. This PR fixes that, so that only users with proper permissions to see those pages will have the appropriate options shown to them. This change also takes into account our advanced permissions which permits non-admin to make changes to a subset of our settings page.

### How to verify

Without advanced permissions:
- Sign in as a non-admin
- Open the command palette and type "settings"
- You should see no admin settings links

With advanced permissions:
- Sign in as admin
- Go to permissions page and select "Application" tab
- Give all users access to settings
- Sign out then sign in as a non-admin
- Open the command palette and type "settings"
- You should see some settings pages but not all (General settings should be available, but Setup settings should not be)

### Checklist
